src/tests/test_piranha_java.rs

```diff
@@ -96,6 +96,39 @@ fn test_user_option_delete_consecutive_lines() {
   _helper_user_option_delete_consecutive_lines(_path, true);
 }
 
+#[test]
+fn test_new_line_character_used_in_string_literal() {
+  initialize();
+  let path_to_scenario = PathBuf::from("test-resources")
+    .join(JAVA)
+    .join("new_line_character_used_in_string_literal");
+
+  let piranha_arguments = piranha_arguments! {
+    path_to_configurations = path_to_scenario.join("configurations").to_str().unwrap().to_string(),
+    language = PiranhaLanguage::from(JAVA),
+    dry_run= true,
```

Comment:
```suggestion
    dry_run = true,
```

I am still fairly OCD-adjacent about these equals whitespace issues.